docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -82,9 +82,8 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 
 ## Server (`server`)
 
-- `PROTOCOL`: **http**: \[http, https, fcgi, unix\]
-- `DOMAIN`: **localhost**: Domain name of this server.
 - `ROOT_URL`: **%(PROTOCOL)s://%(DOMAIN)s:%(HTTP\_PORT)s/**: Full public URL of Gitea server.
```

Comment:
If we break config we should do it rigth and maybe rename variable. 
Like using more self-explaining name PUBLIC_URL=https://try.gitea.io and LISTEN_ADDR=unix:///var/run/gitea.sock LISTEN_ADDR=https://0.0.0.0:443

A breaking note should follow but that way admin know what is the impact of config. 
Maybe we can also try to migrate the config.

Reply:
@sapk I like your idea. But this maybe break the old config.

Reply:
Your change will break thing in the (rare ?) case if ROOT_URL is not defined in the config file.

Reply:
I will send another commit to this PR.